README.md

```diff
@@ -54,6 +54,10 @@ Key features of the application.
 - A user can create a purchase for a group
 - User authentication
 - Splash page
+- User can view all purchases
+- User can view all groups
+- User can view all purchases for a group
+- Login and signup links on splash page
 
```

Comment:
As per the project [description](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=about%20building%20a-,mobile%20web%20application,-where%20you%20can) and the given design guidelines, This app should be a mobile app with mobile view, so please consider the following changes:

- [ ] At the categories page(Home page), Please add the correct placement for all page elements (Layout: composition and space between elements) and also add the correct font family, font sizes, and font weight for all elements too to fulfill the project required [design guidelines](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#design).

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/Chimoneg27/Rails-Budget-App/assets/68796232/b9821e9b-1219-4691-a19f-b89a884fdfc5)  | ![image](https://github.com/Chimoneg27/Rails-Budget-App/assets/68796232/2a033cfc-8a0b-4143-a91c-ccb3e9f05564)  |

- [ ] Same here at the transaction page, Please add the correct placement for all page elements (Layout: composition and space between elements) and also add the correct font family, font sizes, and font weight for all elements too to fulfill the project required [design guidelines](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#design).

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/Chimoneg27/Rails-Budget-App/assets/68796232/0a97788a-c49f-47f0-95c7-798abfdc2fa1)  | ![image](https://github.com/Chimoneg27/Rails-Budget-App/assets/68796232/9df53262-5326-463d-ad1a-0051b3786e40)  |

- [ ] When I run the specs you add, It takes forever to run and never return anything (success or failed), Please fix that.

![image](https://github.com/Chimoneg27/Rails-Budget-App/assets/68796232/a6ba198b-019a-484a-ac9b-dc3d5ab0b41d)